tests/foreman/ui_airgun/test_puppetenvironment.py

```diff
@@ -0,0 +1,97 @@
+from pytest import raises
+from nailgun import entities
+import pytest
+from robottelo.datafactory import (
+    gen_string,
+)
+from robottelo.decorators import (
+    tier2,
+    run_only_on,
+)
+
+
+@pytest.fixture(scope='module')
+def fixture_for_puppet_environment():
+    """Fixture returns values for new environment"""
+    name = gen_string('alpha')
+    org = entities.Organization().create()
+    location = entities.Location().create()
+    puppetEnvironmentValues = {
+        'environment.name': name,
+        'locations.resources.assigned': [location.name],
+        'organizations.resources.assigned': [org.name],
+    }
+    return puppetEnvironmentValues
+
+
+def test_positive_create(session, fixture_for_puppet_environment):
+    values = fixture_for_puppet_environment
```

Comment:
redundant
rename initial fixture to something smaller like `init_values` or whatever